ops/ccs-ops-misc/synthetic-data/scripts/synthea-automation/README.md

```diff
@@ -0,0 +1,66 @@
+# Synthea Automation Scripts and Docker Image
+
+This directory contains several Synthea-related scripts pertaining to generating synthetic data
+using Synthea, as well as the definition for a Docker image (`bfd-mgmt-synthea-generation`) that
+includes said scripts.
+
+## Generating Synthea Data Locally Using the `bfd-mgmt-synthea-generation` Docker Image
+
+It is _preferred_ to generate Synthea data locally using the `bfd-mgmt-synthea-generation` Docker
+image, as it has the following advantages over using the `run_synthea_locally.sh` script:
+
+1. The Docker image comes pre-built with the necessary mapping files, the Synthea executable, and
+   all other dependencies
+2. Using the Docker image does not affect your local filesystem
+3. Using the Docker image does not require connecting to AWS or any external source; all data
+   generation is done locally to your machine
+
+Usage is simple:
+
+1. Ensure you are authenticated with AWS and are able to pull images from our ECR as well as from S3
```

Comment:
A little unclear, above it mentions one of the benefits is not requiring connecting to AWS, but we do clearly need access to s3 here and below.

Reply:
Yeah, I should probably be more specific as to what I mean by the "Docker image does not require connecting to AWS": the _container_ itself does not reach out to S3 or any other internet-facing service when running Synthea generation -- it is self-contained and the image includes only the dependencies necessary to _run_ Synthea generation and nothing else. *However*, and this is probably the issue, Synthea generation _does_ require the previous `end_state.properties` to properly generate data, and that just so happens to live in S3 right now. If this file is ever moved from S3 to some other store, the _container/image_ would not need to be updated to support that -- only how we _run_ the container would need to change.

Reply:
I've updated this bullet point in the `README` to clarify what I meant by the "Docker image does not require connecting to AWS". Let me know if you feel that clears things up for you, @lsmitchell.

Reply:
Yup, thanks for clarifying!